src/plone/restapi/tests/http-examples/tusreplace_patch.req

```diff
@@ -1,8 +1,7 @@
-PATCH /plone/myfile/@tus-upload/4e465958b24a46ec8657e6f3be720991 HTTP/1.1
+abcdefghPATCH /plone/myfile/@tus-upload/4e465958b24a46ec8657e6f3be720991 HTTP/1.1
```

Comment:
This really looks bad. No idea what happend here.

Reply:
This is left over from a bad test run before I made it seek to the end of the file before writing the body. This whole PR is still a WIP and not really ready for review.

Reply:
@tisto thanks for updating the branch. I will take another look in the next day or two.

Reply:
@davisagli ok, sorry. After @pbauer's email I was under the assumption this would be ready for review. Will wait then...let me know when I can help...

Reply:
@tisto I'm glad you did take a look because I had forgotten that the branch needed to be updated, and it sounds like that may have fixed some of the remaining errors I was seeing.